src/Kiota.Builder/Refiners/RubyRefiner.cs

```diff
@@ -88,24 +92,24 @@ public override void Refine(CodeNamespace generatedCode)
             }
             return null;
         }
-        protected void AddNamespaceModuleImports(CodeElement current, String clientNamespaceName) {
+        protected void AddNamespaceModuleImports(CodeElement current, string clientNamespaceName) {
```

Comment:
nit: You missed a spot on line 99 as well 😄

Reply:
thanks, fixed.